operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -307,8 +310,24 @@ public void createOrRenew(String namespace, String clusterName, Map<String, Stri
             log.info("{}: Certificates renewed", this);
         }
 
-        caCertSecret = secretCertProvider.createSecret(namespace, caCertSecretName, certData, labels, ownerRef);
-        caKeySecret = secretCertProvider.createSecret(namespace, caKeySecretName, keyData, labels, ownerRef);
+        // cluster CA certificate generation annotation handling
+        int caCertGeneration = INIT_GENERATION;
+        if (caCertSecret != null && caCertSecret.getData().get(CA_CRT) != null) {
+            String caCertGenerationAnnotation = caCertSecret.getMetadata().getAnnotations().get(ANNO_STRIMZI_IO_CA_CERT_GENERATION);
+            if (caCertGenerationAnnotation != null) {
+                caCertGeneration = Integer.parseInt(caCertGenerationAnnotation);
+                if (caRenewed) {
+                    caCertGeneration++;
+                }
+            }
+        }
+
+        // TODO: handling cluster CA key generation annotation
```

Comment:
What does this TODO mean?

Reply:
Just a placeholder to remember to handle the key generation annotation, but I can remove it right now.

Reply:
If you need a reminder I think it's better to have an issue. (I know: I'm the worst for TODOs, but please pick me up on this too).